opcua/common/xmlexporter.py

```diff
@@ -0,0 +1,328 @@
+"""
+from a list of nodes in the address space, build an XML file
+format is the one from opc-ua specification
+"""
+import logging
+import xml.etree.ElementTree as Et
+
+from opcua import ua
+from opcua.ua import object_ids as o_ids  # FIXME needed because the reverse look up isn't part of ObjectIds Class
+
+
+class XmlExporter(object):
+
+    def __init__(self, server, node_set_attrs):
+        self.logger = logging.getLogger(__name__)
+        self.etree = Et.ElementTree(Et.Element('UANodeSet', node_set_attrs))
+        self.server = server
+
+    def export_xml(self, node_list, xmlpath):
```

Comment:
somewhere we also need to export address space... or maybe we just get the address space if the nodeid identifier is not 0